vignettes/VectorControl.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Vector Control"
+title: "Vector Control: Bednets"
```

Comment:
Might be helpful to rename the file as `VectorControl_bednets` given that the other is `VectorControl_IRS`